### PR TITLE
20.03 NixOS release 

### PR DESCRIPTION
**Pull request by garbas**
Coordinate with NixOS/nixpkgs#66640

deprecating #297 

Closes #401 by providing the equivalent change.

### CODE REVIEW

File: news.xml [hunk 1]
```diff
@@ -306,14 +352,14 @@
...
+          src="https://d2z6c3c3r6k4bx.cloudfront.net/uploads/event/logo/1005856/banner.png" alt="NixCon logo" /></a>
       We're happy to announce that <strong>NixCon 2015</strong>, the
       first Nix Conference, will take place on <strong>November
-      14—15th 2015 in Berlin</strong>. For more information, see the
+        14—15th 2015 in Berlin</strong>. For more information, see the
       <a href="http://conf.nixos.org">NixCon website</a>. And please
-      consider <a
-      href="http://conf.nixos.org/submit-a-talk.html">submitting a
-      talk</a>!
+      consider <a href="http://conf.nixos.org/submit-a-talk.html">submitting a
```
Comment (davidak) [inline news.xml:361]: why was the identation changed here everywhere? it looks messed up

File: news.xml [hunk 2]
```diff
@@ -444,15 +480,15 @@
...
+        vulnerability</a> has been discovered in OpenSSL. All stable
       NixOS releases prior to version
       <strong>13.10.35708.15a465c</strong> are vulnerable. (You can
       see your current version by running <tt>nixos-version</tt>.) To
       upgrade to the latest NixOS version, run <tt>nixos-rebuild
-      switch --upgrade</tt>. You can verify whether you are safe by
+        switch --upgrade</tt>. You can verify whether you are safe by
       running
 
-<pre class="code">
```
Comment (davidak) [inline news.xml:455]: shouldn't the start pre have the same identation as end pre? the block between could be idented

File: news.xml (hunk 1 above)
Comment (garbas) [reply news.xml:361]: i think my editor did this. need to somehow revert this.

File: news.xml (hunk 2 above)
Comment (garbas) [reply news.xml:455]: i think my editor did this. need to somehow revert this.

**Comment by davidak** [comment]
The rendered pages look good :+1:

File: news.xml [hunk 3]
```diff
@@ -1,6 +1,65 @@
...
+        </tbody>
+      </table>
+      <h3>My Reflections and Closing</h3>
+      <p>
+        Being release manager for 20.03 has been a poignant moment for me in
+        being part of NixOS. I had my goals that I set out before I was
+        appointed, but I was really surprised how respected I am in the
+        community. My primary goal was “work collaboratively with all
+        participants in the NixOS project and being supportive of their
+        efforts”. I feel I ✨ shine best in that dynamic in the project, so
```
Comment (worldofpeace) [inline news.xml:45]: the `best` here should be **bold**

File: news.xml [hunk 4]
```diff
@@ -1,6 +1,65 @@
...
+        this really was perfect for me. I hope releasing NixOS has felt better
```
Comment (worldofpeace) [inline news.xml:46]: the `felt` here should be **bold**

File: news.xml [hunk 5]
```diff
@@ -1,6 +1,65 @@
...
+        for those involved. With the seeds I’ve planted it should continue to
+        bloom this way.
+      </p>
+      <p>
+        I’d like to thank <a hred="https://discourse.nixos.org/u/disassembler">Samuel Leathers</a>,
+        my co release manager, for his proportionate effervescence and guidance;
```
Comment (worldofpeace) [inline news.xml:52]:
```suggestion
        my co release manager, for his congruent effervescence and guidance;
```

this is the work I was actually looking for :smile:

**Comment by worldofpeace** [comment]
I added numbers to https://hackmd.io/Uy24aMgYThyPIHZWyXGRcA?both

File: news.xml [hunk 6]
```diff
@@ -1,6 +1,65 @@
...
+            <th>Author</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>TODO</td>
+            <td>TODO</td>
+          </tr>
+        </tbody>
+      </table>
```
Comment (worldofpeace) [inline news.xml:37]: this section is replaced in my markdown at https://hackmd.io/Uy24aMgYThyPIHZWyXGRcA?both
"## NixOS 20.03 Contributors ### Top 30 ordered by commits [TABLE]`

**Comment by worldofpeace** [comment]
When I look at the deployed website I see 19.09 download links, and the download page says 19.09. The flake is updated and so has the channel. I'm not sure what I'm missing.

**Comment by worldofpeace** [comment]
The final tweaks from IRC, just waiting now...

**Comment by worldofpeace** [comment]
Thanks @globin for helping me out greatly :heart:

File: news.xml [hunk 7]
```diff
@@ -1,6 +1,117 @@
 <?xml version="1.0" encoding="utf-8" ?>
 
 <news>
+  <item>
+    <pubDate>TODO</pubDate>
```
Comment (das-g) [inline news.xml:5]: I guess this should have been replaced by the actual current timestamp right before merging?

**Pull request merged**